subdir/attestationd/src/spdmlite.cpp

```diff
@@ -129,6 +93,46 @@ void intialiseSpdmHandler(SpdmHandler& spdmHandler,
             co_return;
         });
 }
+
+net::awaitable<void> onNeighbhorFound(
+    net::io_context& io_context,
+    std::shared_ptr<sdbusplus::asio::connection> conn,
+    sdbusplus::asio::object_server& dbusServer, SpdmHandler& spdmHandler,
+    SpdmResponderIface& spdmResponder,
+    std::shared_ptr<SpdmDeviceIface>& spdmDevice, const std::string& remotePort,
+    const boost::system::error_code& ec, const std::string& propVal)
+{
+    LOG_INFO("Neighbour LLDP ManagementAddressIPv4 changed: {}", propVal);
+    SpdmDeviceIface::ResponderInfo responderInfo{"device1", propVal,
+                                                 remotePort};
+    spdmDevice = std::make_shared<SpdmDeviceIface>(conn, dbusServer,
+                                                   responderInfo, spdmHandler);
+    intialiseSpdmHandler(spdmHandler, *spdmDevice, spdmResponder);
+    co_return;
+}
+net::awaitable<void> updateNeighbourDetails(
+    net::io_context& io_context,
+    std::shared_ptr<sdbusplus::asio::connection> conn,
+    sdbusplus::asio::object_server& dbusServer, SpdmHandler& spdmHandler,
+    SpdmResponderIface& spdmResponder,
+    std::shared_ptr<SpdmDeviceIface>& spdmDevice, const std::string& remotePort)
+
+{
+    auto [ec, propVal] = co_await getProperty<std::string>(
+        *conn, LLDP_SVC, std::format(LLDP_REC_PATH, "eth1"), LLDP_INTF,
```

Comment:
Should this "eth1" not be hardcoded?
For Skiboards, it is eth1, but later for P12, it will be "eth2"/"<however they name the internal interface>"